<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projet</class>
 <widget class="QDialog" name="projet">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>482</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create a new project</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Project name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Root directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Existing files or ressources to copy in the project directory</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="addedFilesText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>501</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectRootDirButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rootDirEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="projNameEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addFilesButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>projet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>projet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
